### ISSUE TITLE
DatePickerField doesn't allow to select current date 

### ISSUE THREAD

**Issue by idiidi** [enisn/UraniumUI#743]
When using a DatePickerField on Android and pressing the field to open up the date selector, pressing the current date doesn't populate the field with the date.
Selecting a different date first, pressing ok and then selecting the current date works

**Comment by enisn** [comment]
Seems similar with: https://github.com/enisn/UraniumUI/issues/84

Since native maui DatePicker doesn't allow `null` value. We visually make it null but still, the previously selected value is kept in the encapsulated maui picker

**Comment by idiidi** [comment]
Is there a workaround? 

My issue is more to do with not being able to select the current date, and not the issue with being not being able to select the same date twice. I know this the same issue behind the scenes as the current date is selected but not shown. But being able to select the current date, or even having the current date shown by default and not being blank will be an improvement.

**Comment by StepKie** [comment]
@idiidi Yeah it is annoying since the underlying cause [is still (unfixed) in MAUI repo](https://github.com/dotnet/maui/issues/13156).
We also manually do `public DateTime PropertyBoundToDatePickerField { get; set; } = DateTime.Now` in our viewmodel class...

@enisn I would close these duplicates right away though, so it doesn't seem that there are over 100 open issues in UraniumUI, when there are clearly less.
